stats_test.go

```diff
@@ -33,6 +33,7 @@ func testExporter(opts Options) (*testStatsExporter, error) {
 	if err != nil {
```

Comment:
You could add another option to fix the test noise, e.g.
```go
if opts.OnError == nil {
    opts.OnError = func(_ error) {}
}
```
That should do it, right?

Reply:
yes that worked thanks!!